src/core/tests/any.cpp

```diff
@@ -419,15 +419,15 @@ TEST_F(AnyTests, constructFromVariantImpl) {
 TEST_F(AnyTests, dynamicPointerCastToVariantWrapper) {
     OPENVINO_SUPPRESS_DEPRECATED_START
     Any p = std::make_shared<ngraph::VariantWrapper<std::string>>("42");
-    auto str_variant = std::dynamic_pointer_cast<ngraph::VariantWrapper<std::string>>(p);
+    auto str_variant = ov::Any(p).as<std::shared_ptr<ngraph::VariantWrapper<std::string>>>();
```

Comment:
are we going to remove VariantWrapper itself?
it's also deprecated. Maybe we need to start with it?

Reply:
can ignore this, which will be removed in PR: 15580